linux-anvil-ppc64le-cuda/Dockerfile

```diff
@@ -69,6 +85,21 @@ RUN source /opt/conda/etc/profile.d/conda.sh && \
     chgrp -R lucky /opt/conda && \
     chmod -R g=u /opt/conda
 
+# Download and cache CUDA related packages.
+RUN if [[ "$CUDA_VER" == "9.2" || "$CUDA_VER" == "10.0" || "$CUDA_VER" == "10.1" ]]; then \
+        echo "`cudatoolkit` not available for CUDA_VER<10.2"; \
```

Comment:
Just a nitpick: maybe we should error out here?

Reply:
I think Isuru wants them around because they are being used somewhere.